game/src/main/java/com/github/mrpaulblack/tron/PlayerColor.java

```diff
@@ -18,9 +18,8 @@
 
 	/**
 	 * <h1><i>toString</i></h1>
-	 * <p>This method is converting the ENUM to strings that are supported
-	 * by the MAEDN specification.</p>
```

Comment:
maedn?

Reply:
Taken that out since MAEDN was the protocol we used last semester with the other exam groups. We're using a similar setup here but since I copied some of the enums there were also some comments from last semester that were transfered and that are no longer accurate.

Reply:
Please fix; You can do a search and replace for `MAEDN` to `TRON`